docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
+3. User looks for a stacktrace with their App’s processes name
+4. User finds stacktrace + root cause and begins to fix
+5. User commits fix
+6. User’s Build passes
+7. User merges PR
+#### Use Case
+1. GitHub has Status update and comment from fastlane on PR indicating that the build failed
+2. User opens fastlane ci Web UI
+4. Assuming already logged in
+5. User opens Project for their Repo + branch
+6. User types in their name to filter
```

Comment:
I'm not sure I understand this part, once a project is open, we're in the project, no need to search, right?

Reply:
I think the idea here is that there is potentially a massive list of builds. Maybe we should allow filtering. I think the first think a user might be looking for is the builds that were kicked off on a PR they created. So it might be best to allow them to filter in order to find the Build they are looking for.